testsuite/src/test/java/org/jboss/da/test/server/scm/SCMTest.java

```diff
@@ -46,6 +46,22 @@ public void shouldCloneGitRepository() throws Exception {
         }
     }
 
+    @Test
+    public void shouldBeAbleToCloneGitTag() throws Exception {
+
+        // this test should use the shallow cloning feature
```

Comment:
Please use ``git rev-list HEAD --count`` to find out if it is a shallow clone (has to print 1)